app/views/draws/edit.html.erb

```diff
@@ -1,2 +1,3 @@
+<%= provide(:title, "Edit draw " + @draw.name) %>
```

Comment:
I don't know if we need to have the word `draw` here - on the one hand, it's not going to be required for the draw names, but I'd imagine all admins would include it (e.g. `Sophomore draw 2017`). @esoterik what do you think?